<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>491</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="openFileButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>390</y>
      <width>171</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Открыть файл</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputDateEdit">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>141</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>121</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите дату :</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>40</y>
      <width>141</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="nextDayButton">
       <property name="text">
        <string>Следующий день</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="prevDayButton">
       <property name="text">
        <string>Предыдущий день</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="isLeapButton">
       <property name="text">
        <string>Високосный?</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="weekNumberButton">
       <property name="text">
        <string>Номер недели</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="birthdayButton">
       <property name="text">
        <string>День рождение</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="durationButton">
       <property name="text">
        <string>количество дней</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addDateButton">
       <property name="text">
        <string>Добавить дату</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="editDateButton">
       <property name="text">
        <string>Изменить дату</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>40</y>
      <width>111</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
